lua/smart-splits/mux/utils.lua

```diff
@@ -75,6 +75,13 @@ function M.startup()
       end,
     })
   end
+  if mux.update_layout_details then
+    vim.api.nvim_create_autocmd({ 'VimResume', 'VimEnter', 'VimResized', 'FocusGained' }, {
```

Comment:
This is clever; I assume `VimResized` fires properly when resized, for example, due to opening a mux split. Nice optimization.

Reply:
What happens if you have a keybind that creates a mux split but doesn't focus it, so nvim does not lose focus?

Reply:
I meant, what if you have a mux keybind that does that, not initiated by nvim? Is there a way for nvim to be notified of that?

Reply:
I'm not entirely sure 😅 

I did a quick test by dropping this in my `init.lua` (where `dbg_schedule` is a debug helper I've defined in my config)

```lua
vim.api.nvim_create_autocmd('VimResized', {
  callback = function()
    dbg_schedule('VimResized: resizing splits')
  end,
})
```

and it looks like what I'm describing does properly fire `VimResized` anyway, so regardless, this works great.

And regardless, this is still a net improvement over the status quo today anyway 👍